### DISCUSSION TITLE
Enable native 'external-warm-transfers'?

### DISCUSSION THREAD

**Discussion by nastiazhyrnova** [twilio-professional-services/flex-project-template#656] [Q&A]
Hi,
I am trying to enable external warm transfer in "custom transfer directory" feature, but it doesn't let me use it unless "Conference" feature or "native external warm transfer" is enabled: 

<img width="881" alt="Screenshot 2024-10-22 at 11 22 04" src="https://github.com/user-attachments/assets/78746fde-b9e1-4e57-bb47-d7c3d22ac270">

https://github.com/twilio-professional-services/flex-project-template/blob/f0ec43d14f1d3a769a683232ef6516557e391e9a/plugin-flex-ts-template-v2/src/feature-library/custom-transfer-directory/config.ts#L34

I have "Warm transfers" enabled under Features:
![Screenshot 2024-10-22 at 11 19 20](https://github.com/user-attachments/assets/4b5dbbef-033a-4117-8f01-3eab6dac442d)

However, it seems to be appearing in store under `flex-warm-transfers` and not under `external-warm-transfers`. Where could I enable this native feature, or is it an error and it refers to `flex-warm-transfers`?

**Comment by dremin** [comment]
The native feature the alert is referring to cannot be enabled manually and is no longer supported, so you will need to enable the conference feature if you would like to use the external warm transfer option.
